eng/restore/dirs.proj

```diff
@@ -17,24 +17,5 @@
     <Project Include="binplacePackages/binplacePackages.depproj" />
   </ItemGroup>
 
-  <!-- Set up project references to use the sdk restore target. -->
-  <Target Name="IncludeProjectReferences" DependsOnTargets="FilterProjects">
-    <ItemGroup>
-      <ProjectReference Include="@(Project)" />
-    </ItemGroup>
-  </Target>
-
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.targets" />
-
-  <!-- Overwrite the SDK build target. -->
-  <Target Name="Build" DependsOnTargets="FilterProjects">
-    <MSBuild Targets="Build"
-             Projects="@(Project)"
-             Properties="BuildConfiguration=$(BuildConfiguration);%(Project.AdditionalProperties)"
-             BuildInParallel="true"
-             ContinueOnError="ErrorAndStop" />
-
-    <!-- Given we ErrorAndContinue we need to propagate the error if the overall task failed -->
-    <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
-  </Target>
+  <Import Project="$(RepositoryEngineeringDir)dir.traversal.targets" />
```

Comment:
Why did we delete these lines and instead imported a different targets file? The only relevant thing I see on the new imported files is the allowance of targeting 5.0.

Reply:
explained here https://github.com/dotnet/corefx/pull/40049#discussion_r311716483